tabpy-server/tabpy_server/app/util.py

```diff
@@ -31,13 +23,15 @@ def validate_cert(cert_file_path):
 
     https_error = 'Error using HTTPS: '
     if now < not_before:
-        log_and_raise(https_error +
-                      'The certificate provided is not valid until {}.'.format(
-                          not_before), RuntimeError)
+        msg = (https_error +
```

Comment:
If using formatted string literals would be more idiomatic to write this as `f"{https_error}The certificate provided is not valid until {not_before}."`

Reply:
Yes, eventually we want to use formatted strings everywhere as they are easy to read and safer. In this change, I missed some places where the strings can be updated... Some next change maybe :)